classes/entity/enemy/bobomb/bobomb.gd

```diff
@@ -43,13 +46,20 @@ func _physics_step() -> void:
 	if !struck:
 		_update_sprites()
 	
-	if fuse.animation == "lit":
-		fuse_time -= 1
+	if _is_lit:
+		fuse.animation = "lit"
+		var fuse_frame: float = max(0, BUILDUP_SOUND_START - _fuse_time)
+		var fuse_progress: float = fuse_frame / BUILDUP_SOUND_START
```

Comment:
Naming these variables "fuse" implies they're related to the fuse sprite, not the red flash.